docs/examples/reflections/plot_convert_iam_models.py

```diff
@@ -0,0 +1,190 @@
+
+"""
+IAM Model Conversion and Fitting
+================================
+
+Illustrates how to convert from one IAM model to a different model using
+pvlib.iam.convert, and how to fit an IAM model to data using
+pvlib.iam.fit
```

Comment:
I think fitting is distinct enough from converting to warrant a separate example page

Reply:
Agreed!